pypeit/tests/test_spectrographs.py

```diff
@@ -62,6 +62,44 @@ def test_select_detectors_mosaic():
     assert spec.select_detectors(subset=[3,(1,2,3)]) == [3,(1,2,3)], 'Bad detector selection'
 
 
+def test_list_detectors_deimos():
+    deimos = load_spectrograph('keck_deimos')
+    dets = deimos.list_detectors()
+    assert dets.ndim == 2, 'DEIMOS has a 2D array of detectors'
+    assert dets.size == 8, 'DEIMOS has 8 detectors'
+    mosaics = deimos.list_detectors(mosaic=True)
+    assert mosaics.ndim == 1, 'Mosaics are listed as 1D arrays'
+    assert mosaics.size == 4, 'DEIMOS has 4 predefined mosaics'
+
+
+def test_list_detectors_mosfire():
+    mosfire = load_spectrograph('keck_mosfire')
+    dets = mosfire.list_detectors()
+    assert dets.ndim == 1, 'MOSFIRE has a 1D array of detectors'
+    assert dets.size == 1, 'MOSFIRE has 1 detector'
+    with pytest.raises(PypeItError):
+        mosaics = mosfire.list_detectors(mosaic=True)
+
+
+def test_list_detectors_mods():
+    mods = load_spectrograph('lbt_mods1r')
+    dets = mods.list_detectors()
+    assert dets.ndim == 1, 'MODS1R has a 1D array of detectors'
+    assert dets.size == 1, 'MODS1R has 1 detector'
+    with pytest.raises(PypeItError):
+        mosaics = mods.list_detectors(mosaic=True)
+
+
+def test_list_detectors_hires():
+    hires = load_spectrograph('keck_hires')
+    dets = hires.list_detectors()
+    assert dets.ndim == 1, 'HIRES has a 1D array of detectors'
+    assert dets.size == 3, 'HIRES has 1 detector'
```

Comment:
Did you want to say "HIRES has 3 detectors"?

Reply:
Yes!  Good catch.  Fixed.